src/genomedata_load_data.c

```diff
@@ -117,6 +117,42 @@ typedef struct {
 #endif /* GCC_VERSION >= 20500*/
 #endif /* __GNUC__ */
 
+void load_data(char* gdfilename, char* trackname, bool verbose);
+
+static PyObject* load_data_from_stdin(PyObject* self, PyObject* args) {
+  const char* gdfilename;
+  const char* trackname = "dummy";
+  int verbose; /* needs to be int, not bool for parsing */
+
+  if (!PyArg_ParseTuple(args, "ssp", &gdfilename, &trackname, &verbose)) /* string string bool predicate */
+    return NULL;
+
+  load_data(gdfilename, trackname, (bool)verbose);
+
+  /* Return None in Python */
+  Py_RETURN_NONE;
+}
+
+static PyMethodDef LoadDataMethods[] = {
+    {"load_data_from_stdin", load_data_from_stdin, METH_VARARGS,
+     "Loads data in BED or WIG format from stdin into a track in a genomedata archive"},
+    {NULL, NULL, 0, NULL} /* Sentinel */
+};
+
+static struct PyModuleDef loadDateModule = {
```

Comment:
why camelCase?

Reply:
This all needs to go anyway. I think there must have been some unfortunate undo before the commit.